prog/weaver/main.go

```diff
@@ -69,6 +71,7 @@ func main() {
 	flag.IntVar(&bufSzMB, "bufsz", 8, "capture buffer size in MB")
 	flag.StringVar(&httpAddr, "httpaddr", fmt.Sprintf(":%d", weave.HTTPPort), "address to bind HTTP interface to (disabled if blank, absolute path indicates unix domain socket)")
 	flag.StringVar(&iprangeCIDR, "iprange", "", "IP address range to allocate within, in CIDR notation")
+	flag.StringVar(&defltSubnet, "defaultsubnet", "", "default subnet for IP address allocation, in CIDR notation")
```

Comment:
I consider 'universe' to be a poor name, and said so during my IPAM review.  It's even worse in the context of multi-subnet IPAM.  But while it is confined to identifiers in the code, at least there is the prospect that it will be replaced with a better name at some point.  Please let's not use it in a user-visible context.